.gitignore

```diff
@@ -8,3 +8,4 @@ default.pd
 tags
 /.retools/
 default*
+!defaults.go
```

Comment:
can we change `default*` to `default.pd*`?

Reply:
Good idea.